packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -41,9 +43,19 @@ export interface ViewCustomTimings {
 export const THROTTLE_VIEW_UPDATE_PERIOD = 3000
 export const SESSION_KEEP_ALIVE_INTERVAL = 5 * ONE_MINUTE
 
-export function trackViews(location: Location, lifeCycle: LifeCycle) {
+export function trackViews(location: Location, lifeCycle: LifeCycle, newLocationListener?: NewLocationListener) {
```

Comment:
You could set the default value directly in the signature, so you don't have two names for the same thing (`onNewLocation` / `newLocationListener`):
```suggestion
export function trackViews(location: Location, lifeCycle: LifeCycle, onNewLocation: NewLocationListener = noop) {
```